observability-charter.md

```diff
@@ -0,0 +1,207 @@
+# CNCF SIG Observability Charter
+
+- [CNCF SIG Observability Charter](#cncf-sig-observability-charter)
+  - [Introduction](#introduction)
+  - [Mission](#mission)
+  - [Areas considered in Scope](#areas-considered-in-scope)
+  - [Areas considered out of Scope](#areas-considered-out-of-scope)
+  - [Roadmap & Initial Efforts](#roadmap--initial-efforts)
+  - [Governance](#governance)
+  - [Operations](#operations)
+
+*Initially authored by [Matthew Young][matthew young] with grateful review and
+contributions from:
+[Alex Nauda][Alex Nauda],
+[Alois Reitbauer][Alois Reitbauer],
+[Bartłomiej (Bartek) Płotka][Bartłomiej (Bartek) Płotka],
+[Daniel Khan][Daniel Khan],
+[Daniel Prata][Daniel Prata],
+[Lincoln Sward][Lincoln Sward],
+[Matthias Loibl][Matthias Loibl],
+[Michael Hausenblas][Michael Hausenblas],
+[Ricardo Aravena][Ricardo Aravena],
+[Richard Hartmann][Richard Hartmann],
+[Sergey Kanzhelev][Sergey Kanzhelev],
+[Steve Flanders][Steve Flanders],
+[Ted Young][Ted Young],
+[Tigran Najaryan][Tigran Najaryan],
+[Tommy Chong][Tommy Chong],
+and [Umair Ishaq][Umair Ishaq].*
+
+<!-- TODO: please put github names here -->
+[Matthew Young]:              https://github.com/halcyondude
+[Alex Nauda]:                 @
+[Alois Reitbauer]:            https://github.com/aloisreitbauer
+[Bartłomiej (Bartek) Płotka]: https://github.com/bwplotka
+[Daniel Khan]:                @
+[Daniel Prata]:               @
+[Lincoln Sward]:              @
+[Matthias Loibl]:             https://github.com/metalmatze
+[Michael Hausenblas]:         https://github.com/mhausenblas
+[Ricardo Aravena]:            https://github.com/raravena80
+[Richard Hartmann]:           https://github.com/RichiH
+[Sergey Kanzhelev]:           @
+[Steve Flanders]:             https://github.com/flands
+[Ted Young]:                  @
+[Tigran Najaryan]:            @
+[Tommy Chong]:                https://github.com/techietommy
+[Umair Ishaq]:                https://github.com/umairishaq
+
+## Introduction
+
+This document describes the purpose and operations of the Cloud Native
+Computing Foundation ([CNCF]) Special Interest Group ([SIG]) on Observability.
+
+This [SIG] focuses on topics pertaining to the observation
+of [cloud native][cn-def] workloads. Additionally, it produces supporting
+material and best practices for end-users and provides guidance and
```

Comment:
I'd still prefer "good practices" over "best practices". Software engineering (not even talking about cloud native yet) is in itself a very young discipline. While established disciplines such as civil engineering benefit from centuries of insights what (absolutely) worked and what not, they are indeed in a position to declare a certain practice as the "best". I do, with all optimism mustered, not believe we are in this position. We can, however, identify and describe good practices: "this worked for X in context Y". WDYT?

Reply:
My 2 cents:
* I believe that it's super tiny detail (: 
* I am not a native English speaker, but I never heard the term "good" practice in this context. We always say "best practices". It does not mean "super super optimal and for sure the best, and there is no way there is better" :upside_down_face: 
* I believe we still aim for "best" practices even in the sense you described. I think about it this way: We are not the smartest in the world, but **to our knowledge** the X is the best practice in context of Y. If you think otherwise, tell us - we **would love to learn**. If the arguments win we move to Z, which will become new best practices! (: Can't see anything wrong with aiming for the best. 

My vote is to leave `best` as we are always aiming for best (best to our, CNCF and community knowledge).